packages/google_maps_flutter/google_maps_flutter/test/google_map_test.dart

```diff
@@ -587,4 +588,34 @@ void main() {
 
     expect(platformGoogleMap.buildingsEnabled, true);
   });
+
+  testWidgets(
+    'Default Android widget is AndroidView',
+    (WidgetTester tester) async {
+      await tester.pumpWidget(
+        const Directionality(
+          textDirection: TextDirection.ltr,
+          child: GoogleMap(
+            initialCameraPosition: CameraPosition(target: LatLng(10.0, 15.0)),
+          ),
+        ),
+      );
+
+      expect(find.byType(AndroidView), findsOneWidget);
+    },
+  );
+
+  // TODO(bparrishMines): Uncomment once https://github.com/flutter/plugins/pull/4017 has landed.
```

Comment:
Why is it required in this order?

Reply:
The tests for the widget are in `google_maps_flutter`, but the changes to the code are in `google_maps_flutter_platform_interface`. However, `google_maps_flutter` pulls the latest version of the platform interface from pub.